lib/client-assets.php

```diff
@@ -1413,10 +1365,8 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	( function() {
 		var editorSettings = %s;
 		window._wpLoadGutenbergEditor = new Promise( function( resolve ) {
-			wp.api.init().then( function() {
```

Comment:
Pretty happy about this one :)

Reply:
Next to eliminate `domReady` and the promise altogether 😉